Sources/FunctionalTableData/TableView/FunctionalTableData+UITableViewDataSource.swift

```diff
@@ -31,8 +31,10 @@ extension FunctionalTableData {
 			let row = indexPath.row
 			let cellConfig = sectionData[row]
 			let cell = cellConfig.dequeueCell(from: tableView, at: indexPath)
-			cell.accessibilityIdentifier = ItemPath(sectionKey: sectionData.key, itemKey: cellConfig.key).description
-			
+			let accessibilityIdentifier = ItemPath(sectionKey: sectionData.key, itemKey: cellConfig.key).description
+			var accessibility = cellConfig.accessibility ?? Accessibility(identifier: accessibilityIdentifier)
```

Comment:
Could we make `accessibility` default to `Accessibility()` rather than `nil`, to match what we do with `CellActions`? Effectively we're forcing each cell to have one anyway.